src/components/LastPost/ListDocs.styles.js

```diff
@@ -0,0 +1,88 @@
+import { makeStyles } from "@material-ui/styles";
+export const useStyles = makeStyles(() => ({
+    container: {
+        background: "#F7F8FC" ,
```

Comment:
Nice good catch but, you should use the color from the theme palette in this case 
`
makeStyles((theme) => ({ 
container: { 
           backgroundColor: theme.palette.primary.main, 
           border: "1px solid #dfe0eb", borderRadius: 3,
            ...
`